examples/c/ex_backup_block.c

```diff
@@ -387,9 +387,11 @@ take_incr_backup(WT_SESSION *session, int i)
                     first = false;
                 }
 
-                error_sys_check(lseek(rfd, (wt_off_t)offset, SEEK_SET));
+                if (lseek(rfd, (wt_off_t)offset, SEEK_SET) == -1)
```

Comment:
I'd repeat the explanatory comment you added in `format/test/backup.c` to here to make sure people reading the example code don't get confused.

Reply:
Fixed.